torch/nn/modules/transformer.py

```diff
@@ -189,9 +190,32 @@ def __init__(self, encoder_layer, num_layers, norm=None, enable_nested_tensor=Tr
         self.layers = _get_clones(encoder_layer, num_layers)
         self.num_layers = num_layers
         self.norm = norm
-        self.enable_nested_tensor = enable_nested_tensor
+        self.use_nested_tensor = enable_nested_tensor
```

Comment:
This change would have BC implications, would be best not to change the name unless absolutely necessary

Reply:
Yes... there's a tradeoff.  It has a changed semantic meaning, so I figure giving it the same name as the init argument might confuse the meaning.  (We sorta have that after I aligned the is_causal name between nn.Transformer* and nn.MHA with SDPA's name is_causal, but it's a hint for nn.MHA/nn.Transformer*)